src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRMessage.java

```diff
@@ -33,48 +39,61 @@ public GitHubPRMessage(String content) {
         this.content = content;
     }
 
-    public String expandAll(AbstractBuild<?, ?> build, TaskListener listener) {
-        String content = getContent();
-        if (content == null || content.length() == 0) {
-            return content; // Do nothing for an empty String
+    /**
+     * Expand all what we can from given run
+     */
+    @Restricted(NoExternalUse.class)
+    @CheckForNull
+    public String expandAll(Run<?, ?> run, TaskListener listener) throws IOException, InterruptedException {
+        return expandAll(getContent(), run, listener);
+    }
+
+    /**
+     * Expand all what we can from given run
+     */
+    @Restricted(NoExternalUse.class)
+    @CheckForNull
+    public static String expandAll(String body, Run<?, ?> run, TaskListener listener)
+            throws IOException, InterruptedException {
+        if (body == null || body.length() == 0) {
+            return body; // Do nothing for an empty String
         }
-//        // Expand environment variables
-//        String s = build.getEnvironment(listener).expand(content);
-        // Expand build variables
-        content = Util.replaceMacro(content, build.getBuildVariableResolver());
 
-        Jenkins jenkins = Jenkins.getInstance();
-        try {
-            ClassLoader uberClassLoader = Jenkins.getInstance().pluginManager.uberClassLoader;
+        // Expand environment variables
+        body = run.getEnvironment(listener).expand(body);
```

Comment:
@lanwen that's not about field hide, that's about reusage of variable that is correct